CHANGELOG.md

```diff
@@ -3,6 +3,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+
+### Added
+- Support for docker private registry - see [#10](https://github.com/ExpediaInc/apiary-authorization/issues/10).
+
+## [2.0.1] - 2019-03-12
+
+### Added
+- Pin module to use `terraform-aws-provider v1.60.0`
```

Comment:
Was this already released? The `version.tf` with that version is in this PR?

Reply:
yes this was released here https://github.com/ExpediaInc/apiary-authorization/releases/tag/v2.0.1

I have just reformatted the code. functioanlity remain same.